tasties/urls.py

```diff
@@ -24,5 +24,7 @@
     path('', views.recipes, name="recipes"),
     path('login/', views.login_user, name="login"),
     path('register/', views.register, name="register"),
-    path('logout/', views.logout_user, name="logout")
+    path('logout/', views.logout_user, name="logout"),
+    # path('single_recipe/<int:recipe_id>/', views.single_recipe, name="single_recipe"),
+    path('single_recipe/', views.single_recipe, name="single_recipe")
```

Comment:
I think `single_recipe` might not be the most descriptive name for the page. Maybe consider `view_recipe`?